crp_APL/controllers/LQR_on_Rails/scripts/lqr_controller.py

```diff
@@ -0,0 +1,277 @@
+#!/usr/bin/env python3
+
+import numpy as np
+import scipy.linalg as la
+import time
+import rclpy
+from rclpy.node import Node
+from autoware_control_msgs.msg import Lateral,Longitudinal
+from autoware_planning_msgs.msg import Trajectory
+from crp_msgs.msg import Ego
+
+# === Parameters =====
+
+# LQR parameter
+lqr_Q = np.eye(5)
+lqr_R = np.eye(2)
+
+lqr_Q[0, 0] = 0.001
```

Comment:
these should also be parameters I think. so that we can easily tune it on the track.